### ISSUE TITLE
Price of book in Euros

### ISSUE THREAD

**Issue by giannispeciale** [jazzsequence/book-review-library#24]

**Comment by jazzsequence** [comment]
Currency should be changeable, so this will need 2 options -- one that will turn the prices on or off and one that can change the currency (maybe this second field can be disabled until the first one is active)

CMB2 has a text_money field that can be used for entering in the price: https://github.com/WebDevStudios/CMB2/wiki/Field-Types#text_money

**Comment by giannispeciale** [comment]
I agree with the two option. Since what I need is for a publishing house site, I attached the ideal scheme of I need to get with the book-review-library plugin.
About CMB2, if I understand, it need to be able to write code and this is very dangerous for me.
 ![book-review-library](https://cloud.githubusercontent.com/assets/10453307/5714475/aeb34a22-9ac9-11e4-9683-cc1e9b43abbe.jpg)

**Comment by jazzsequence** [comment]
No, from a user-perspective, all you'd need to do is activate the option, set the currency symbol and then enter the numeric value into the book review when you're creating it. 

CMB2 is a library that's being included in the plugin in this update to power the forms for book meta information and taxonomies.

**Comment by giannispeciale** [comment]
Great! Thank you!
